### ISSUE TITLE
Be able to get available audio/video devices before starting the video session

### ISSUE THREAD

**Issue by revox-gh** [aws/amazon-chime-sdk-ios#255]
We would like to show a setup screen prior to starting the actual session, so the user can select an appropriate audio- and video device. Maybe I am wrong, but it seems like this is not possible so far, because we need the meeting session and the meeting session needs to be created from a configuration that requires the server request to start the session.

This would be a great addition to the SDK and make the setup process more convenient. Thank you!

**Comment by hokyungh** [comment]
Thank you for opening the issue. Just to clarify, you would like to select audio/video device without creating `MeetingConfiguration` or could you show the screen after creating `MeetingSession`?

If it is latter, we have [DeviceSelectionViewController](https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDKDemo/AmazonChimeSDKDemo/DeviceSelectionViewController.swift#L12).

I'll add enhancement label assuming former, but correct me if I am wrong.

**Comment by hokyungh** [comment]
Actually there is another ticket https://github.com/aws/amazon-chime-sdk-ios/issues/223 with same request. Will close this one.
